FileDelete/FileDelete/FileDelete.vcxproj

```diff
@@ -1,163 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Why did you delete this file?

Reply:
This kind of file would be changed as soon as I open the .sln file via VS 2022. So I would like to remove the change in pull request.